Week 2 changes

denzilbilson opened the pull request:
Set up basic file layout

denzilbilson committed:
Modified README.md to point to new ownership of repo

denzilbilson committed:
set files to represent basic site layout \ add divs to HTML \ add style elements to CSS to define divs

peap commented on portfolio/src/main/webapp/index.html:
```diff
@@ -1,18 +1,35 @@
+    <header>
+        <nav class="menu">
+            <ul>
+                <li><a href ="#home">Home</a></li>
```
HTML tag attributes like `href` here are normally written without any spaces between the attribute name, equals sign, and attribute values. So, this should be `href="#home"` rather than `href ="#home"`.

peap commented on portfolio/src/main/webapp/index.html:
```diff
@@ -1,18 +1,35 @@
+        </nav>
+    </header>
+        <div class="content">
+            <div id="home">
```
Alternatively, check out the `section` tag, which is a semantic way to represent different sections of an HTML page.

peap commented on portfolio/src/main/webapp/index.html:
```diff
@@ -1,18 +1,35 @@
-  </body>
+    <head>
+        <meta charset="UTF-8">
+        <title>My Portfolio</title>
```
Just FYI, you're welcome to change this to `Denzil's Portfolio` or similar, or keep it as-is. Up to you.

peap commented on portfolio/src/main/webapp/style.css:
```diff
@@ -1,9 +1,52 @@
+    padding: 0.5%;
+}
+
+.menu ul li:hover {
```
This is a nice effect 👍

peap commented on portfolio/src/main/webapp/index.html:
```diff
@@ -1,18 +1,35 @@
+    </head>
+    <body>
+    <header>
+        <nav class="menu">
```
If this will be a unique element on the site, like I suspect it will, I suggest making it an ID rather than a class. Addressing IDs in CSS is easier and quicker for the browser to do, compared to looking for classes on all elements.

peap commented on portfolio/src/main/webapp/style.css:
```diff
@@ -1,9 +1,52 @@
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+    margin-left: auto;
```
`#content` matches an element with an ID of `content`, but the div in your HTML that I think you are trying to address only has a class called `content`, so these rules aren't being applied; either the HTML or CSS should change.

peap commented on portfolio/src/main/webapp/style.css:
```diff
@@ -1,9 +1,52 @@
+#home{
+    background: red;
+    height: 100vh;
+    width: 100vw;
```
My version of Chrome adds an 8pm margin for the `body` element when I view your page, so a width of `100vw` means that get the tiniest horizontal scroll bars. I'd experiment with different width units if you're experiencing the same thing. (I think your intent is to take up the full width of the user's browser, but no more.)

denzilbilson committed:
-change title to reflect user
-change classes to id's
-fix spacing issues with href element
-change divs in #content to sections
-remove redundant #content ruleset in style.css
-change width of sections to 99vw to fix horizontal scroll bars

denzilbilson-google commented:
Requested changes have been resolved.

peap commented on portfolio/src/main/webapp/style.css:
```diff
@@ -1,52 +1,60 @@
-.menu ul {
+*{
```
This matches every element on the page, including the root html and body elements; if you just wanted to change one of those, I'd make this rule more specific. If you want all elements to have these settings, that's fine too.

peap reviewed (approved):
Just a quick comment about commit messages. You can see how github (and other interfaces) display the first line of the commit message like an email subject line, so it's useful to make that first line of the message a good summary of what the commit does. (vs. your 859655a commit, which starts with a bulleted list and thus has an implied summary of "-change title to reflect user", which only partially describes the commit.

A better message for that commit messages would be something like:

> Make the requested changes from PR #1 
>
> The changes were:
> - thing one
> - thing two
> ...

So in general, you want a short summary, followed by a blank line, followed by more details about the commit, if warranted.
